Warning when packaging

Running `npm run package` in the current `master` (29a3cd66) gives a warning:

```
WARNING in ./node_modules/adm-zip/util/fileSystem.js 5:18-40
Module not found: Error: Can't resolve 'original-fs' in '/Users/ageorgou/oracc/nisaba/node_modules/adm-zip/util'
 @ ./node_modules/adm-zip/util/index.js 2:0-51
 @ ./node_modules/adm-zip/adm-zip.js 1:14-31
 @ ./src/server/mime.ts 5:15-33
 @ ./src/server/messages.ts 14:15-32
 @ ./src/extension.ts 14:19-47

1 warning has detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.
```

This comes from `webpack`. It has to do with the `original-fs` package, which is an Electron-only package that replicates the built-in Node `fs`. See the same issue [reported on adm-zip's tracker](https://github.com/cthackers/adm-zip/issues/242). 

This *should* be okay. It makes sense that it happens, because webpack doesn’t know about `original-fs` . However, the final code is going to run inside a VS Code instance, which will have `original-fs`.

(What is not clear is why this only happened with merging #54 and not #61, which first used webpack (and was merged first). Maybe the former added extra calls to `adm-zip` or updated the version of `webpack`...)

Even fi there is no problem, the warning is distracting. We could either amend our own dependencies (#63) or possibly suppress it through [webpack's options](https://webpack.js.org/configuration/stats/#statswarningsfilter). I'm leaning towards the latter.

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -10,32 +10,38 @@ const config = {
 
     entry: './src/extension.ts', // the entry point of this extension, 📖 -> https://webpack.js.org/configuration/entry-context/
     output: {
-	// the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
-	path: path.resolve(__dirname, 'dist'),
-	filename: 'extension.js',
-	libraryTarget: 'commonjs2',
-	devtoolModuleFilenameTemplate: '../[resource-path]'
+        // the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'extension.js',
+        libraryTarget: 'commonjs2',
+        devtoolModuleFilenameTemplate: '../[resource-path]'
     },
     devtool: 'source-map',
     externals: {
-	vscode: 'commonjs vscode' // the vscode-module is created on-the-fly and must be excluded. Add other modules that cannot be webpack'ed, 📖 -> https://webpack.js.org/configuration/externals/
+        vscode: 'commonjs vscode' // the vscode-module is created on-the-fly and must be excluded. Add other modules that cannot be webpack'ed, 📖 -> https://webpack.js.org/configuration/externals/
     },
     resolve: {
-	// support reading TypeScript and JavaScript files, 📖 -> https://github.com/TypeStrong/ts-loader
-	extensions: ['.ts', '.js']
+        // support reading TypeScript and JavaScript files, 📖 -> https://github.com/TypeStrong/ts-loader
+        extensions: ['.ts', '.js']
     },
+    ignoreWarnings: [
+        {
+            module: /adm-zip\/util\/fileSystem\.js/,
+            message: /Can't resolve 'original-fs'/,
+        },
+    ],
     module: {
-	rules: [
-	    {
-		test: /\.ts$/,
-		exclude: /node_modules/,
-		use: [
-		    {
-			loader: 'ts-loader'
-		    }
-		]
-	    }
-	]
+        rules: [
+            {
+                test: /\.ts$/,
+                exclude: /node_modules/,
+                use: [
+                    {
+                        loader: 'ts-loader'
+                    }
+                ]
+            }
+        ]
     }
 };
 module.exports = config;

